src/components/SlowPageAlert/index.tsx

```diff
@@ -23,24 +21,29 @@ const SlowPageAlertBase = ({
   allowSlowPagesByDefault,
   getLinkText,
   getMessage,
+  history,
```

Comment:
Do we have the `withRouter` available in this project? That might be time to use it since we need both `history` and `location` in this component.

Reply:
Good idea. This also eliminates the issue I had with needing a fake `history` object in the story.